apps/okreads/browser/src/app/app.component.html

```diff
@@ -23,7 +23,7 @@ <h1>okreads</h1>
     <div class="reading-list-container" data-testing="reading-list-container">
       <h2>
         My Reading List
-        <button mat-icon-button aria-label="close" (click)="drawer.close()">
+        <button data-testing = "close-reading-list" mat-icon-button aria-label="close" (click)="drawer.close()">
```

Comment:
I would remove the spaces around the `=` sign after your `data-*` attributes.

Reply:
@mandarini: Review addressed, please take a look.